pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/entities/Badge.java

```diff
@@ -0,0 +1,79 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.entities;
+
+import javax.persistence.*;
```

Comment:
This star import doesn't even make sense. You're nowhere close to using everything from that package.

Reply:
Idea must have generated this by her own.